ToDoApp/ToDoApp/Assets.xcassets/Button.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Colours should be named the same as in Figma. This convention helps to avoid confusion and clearly understand what colour is expected.
In this case, instead of the name "Button", it should be...🤔 well, I didn't get what the colour is expected to be. For the Text it should be named `primaryText`, TextField should be `secondaryAccent`, ToolBar should be `primaryAccent` etc